UnityProject/Assets/Scripts/UI/Lobby/GUI_LobbyDialogue.cs

```diff
@@ -370,5 +370,14 @@ void HideAllPanels()
 			loggingInPanel.SetActive(false);
 			connectionPanel.SetActive(false);
 		}
+		
+		void Update()
+		{
+			// Log in when the enter key gets pressed
+			if (Input.GetKeyDown(KeyCode.Return) || Input.GetKeyDown(KeyCode.KeypadEnter))
```

Comment:
Only problem with this is that if you press enter on any other screen in the lobby windows, it'll call that Login method. You need to determine when the Account login screen is active. Also it might be a better idea to separate the "enter to continue" behaviour into its own component like the tab next component found here: https://github.com/unitystation/unitystation/tree/develop/UnityProject/Assets/Scripts/UI/GUI/Components 

You will need to make the component know when it is visible (or just put it on the last field of Account login as it will be disabled on the next screens anyway, preventing the behaviour for working) and then you'll need a public UnityEvent to invoke when the user does hit enter so that you can add the OnLogin method to it via the inspector. Don't forget to save it to the lobby scene and commit the scene also.

Reply:
Alright, I will change that. Thanks for the help!